tabs/hide-tabs-macos.css

```diff
@@ -17,18 +17,30 @@
   display: block;
   position: absolute;
   visibility: visible;
+}
+
+/* Reposition the close/maximize/minimize buttons for the "normal" toolbar density  */
+/* When the UI density is set to "normal", the `uidensity` attribute is undefined.  */
+/* `window.document.documentElement.getAttribute("uidensity")` will be null.        */
+:root:not([uidensity]) #TabsToolbar .titlebar-buttonbox-container {
```

Comment:
I guess this may break touch density as well? (I'm not sure if it ever worked with touch density)

Reply:
I don't think the that the touch density is available on macOS, at least not on my machine. That would make sense, since there's no other touchscreen macOS devices.